### PR TITLE
Jbtm 3699 & jbtm 3158

### PR DESCRIPTION
**Pull request by jmfinelli**
This PR migrates `jta-and-hibernate-standalone` to jakarta and addresses JBTM-3158

### CODE REVIEW

File: jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java [hunk 1]
```diff
@@ -182,7 +179,7 @@ private void assertEntities(QuickstartEntity... expected) throws Exception {
     }
 
     private List<QuickstartEntity> getEntitiesFromTheDatabase() throws Exception {
-        DataSource dataSource = InitialContext.doLookup("java:/quickstartDataSource");
+        DataSource dataSource = InitialContext.doLookup(TransactionalConnectionProviderTest.DATASOURCE_JNDI);
         Connection connection = dataSource.getConnection(TransactionalConnectionProvider.USERNAME,
```
Comment (marcosgopen) [inline jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java:183]: What about replacing TransactionalConnectionProvider with TransactionalConnectionProviderTest in this line @jmfinelli ?

**Review by marcosgopen — COMMENTED** [review]
Thanks @jmfinelli for this PR!

File: jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java (hunk 1 above)
Comment (jmfinelli) [reply jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java]: Thanks @marcosgopen!

**Comment by marcosgopen** [comment]
Thanks @jmfinelli for addressing my comments! I therefore approve your PR.

**Pull request merged**